cabal-testsuite/PackageTests/LinkerOptions/NonignoredConfigs/cabal.test.hs

```diff
@@ -51,58 +51,58 @@ linkConfigFlags Dynamic =
 lrun :: [Linking]
 lrun = [Static, Dynamic, Static, Dynamic]
 
-main = cabalTest $ do
+main = do
     -- Skip if on Windows, since my default Chocolatey Windows setup (and the CI
     -- server setup at the time, presumably) lacks support for dynamic builds
     -- since the base package appears to be static only, lacking e.g. ‘.dyn_o’
     -- files.  Normal Windows installations would need support for dynamic
     -- builds, or else this test would fail when it tries to build with the
     -- dynamic flags.
-    skipIfWindows
```

Comment:
For the reviewer: I could undo this indentation, I just change `skipIfWindows` for `skipIfNoSharedLibraries`

Reply:
Isn't the `'do` in `main = do` now spurious, though? GHC should warn about that, I think?

Reply:
Removed